app/mainscreen/page.tsx

```diff
@@ -0,0 +1,9 @@
+import MonthlyView from "./monthlyview/MonthlyView";
```

Comment:
It is not but if i would put it all in page.tsx it will be a some where around 1000 lines.
this way I thought it will be more easily for readability and changing if necessary.
If you prefer it other way i will change it